pyomo/pysp/tests/convert/test_ddsip.py

```diff
@@ -118,52 +106,43 @@ def test_bad_constraint_rhs(self):
     def test_too_many_declarations(self):
         cmd, output_dir = self._get_cmd(
             join(thisdir, "model_too_many_declarations.py"))
-        outfile = output_dir+".out"
-        rc = pyutilib.subprocess.run(cmd, outfile=outfile)
-        self.assertNotEqual(rc[0], 0)
-        self._assert_contains(
-            outfile,
-            b"RuntimeError: Component b.c was "
-            b"assigned multiple declarations in annotation type "
-            b"StochasticConstraintBodyAnnotation. To correct this "
-            b"issue, ensure that multiple container components under "
-            b"which the component might be stored \(such as a Block "
-            b"and an indexed Constraint\) are not simultaneously set in "
-            b"this annotation.")
+        result = subprocess.run(cmd, stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE)
+        self.assertNotEqual(result.returncode, 0)
+        self.assertIn(
+            """RuntimeError: Component b.c was assigned multiple declarations in annotation type StochasticConstraintBodyAnnotation. To correct this issue, ensure that multiple container components under which the component might be stored (such as a Block and an indexed Constraint) are not simultaneously set in this annotation.""".encode('utf-8'),
```

Comment:
Three questions: 
- why did you not make this a `bytes` string like you did on line 81 (instead of unicode / `decode()`), 
- can you restore the previous version where the long line was broken across multiple lines?
- As the new PySP repo has already been created/populated, should we just revert all changes to this file (because they are not likely to be propagated to PySP)?